system/libbase58/base58.c

```diff
@@ -38,7 +39,9 @@ bool b58tobin(void *bin, size_t *binszp, const char *b58, size_t b58sz)
     const unsigned char *b58u = (const unsigned char *)b58;
     unsigned char *binu = (unsigned char *)bin;
     size_t outisz = (binsz + 3) / 4;
-    uint32_t outi[outisz];
+    std::unique_ptr<uint32_t[]> outi(new uint32_t[outisz]);
```

Comment:
This will be less efficient that using alloca.  I doubt it is significant, but how frequently are these functions due to be called?

Reply:
I thing the biggest use would be when LN implements there internal global transaction IDs, if one is passed to a roxie query that calls SOAPCALL or HTTPCALL out.  A rather short base58 id would be generated for each call out.  I'll change to alloca in any case.